sidechain/consensus/aura/src/lib.rs

```diff
@@ -258,19 +303,46 @@ mod tests {
 			timestamp: duration_now(),
 			duration: SLOT_DURATION,
 			ends_at: duration_now() + SLOT_DURATION,
-			parentchain_head: default_header(),
+			last_imported_parentchain_head: default_header(),
 		}
 	}
 
+	fn default_authorities() -> Vec<Public> {
+		vec![
+			Keyring::Alice.public().into(),
+			Keyring::Bob.public().into(),
+			Keyring::Charlie.public().into(),
+		]
+	}
+
+	fn onchain_mock(authorities: Vec<Public>) -> OnchainMock {
+		let validateers = authorities.iter().map(|a| validateer(a.clone().into())).collect();
+		OnchainMock::default().with_validateer_set(Some(validateers))
+	}
+
+	fn onchain_mock_with_default_authorities() -> OnchainMock {
+		onchain_mock(default_authorities())
+	}
+
+	fn create_import_trigger_with_header(
+		header: ParentchainHeader,
+	) -> Arc<TriggerParentchainBlockImportMock<SignedParentchainBlock>> {
+		let latest_parentchain_block =
+			ParentchainBlockBuilder::default().with_header(header.clone()).build_signed();
+		Arc::new(
+			TriggerParentchainBlockImportMock::default()
+				.with_latest_imported(Some(latest_parentchain_block.clone())),
+		)
+	}
```

Comment:
Extracted some common test utility functions to reduce code duplication in tests.

Reply:
The cleanup looks nice in the tests.